cmd/osquery-perf/agent.go

```diff
@@ -135,25 +136,29 @@ func init() {
 }
 
 type Stats struct {
-	startTime              time.Time
-	errors                 int
-	osqueryEnrollments     int
-	orbitEnrollments       int
-	mdmEnrollments         int
-	mdmSessions            int
-	distributedWrites      int
-	mdmCommandsReceived    int
-	distributedReads       int
-	configRequests         int
-	configErrors           int
-	resultLogRequests      int
-	orbitErrors            int
-	mdmErrors              int
-	desktopErrors          int
-	distributedReadErrors  int
-	distributedWriteErrors int
-	resultLogErrors        int
-	bufferedLogs           int
+	startTime                 time.Time
+	errors                    int
+	osqueryEnrollments        int
+	orbitEnrollments          int
+	mdmEnrollments            int
+	mdmSessions               int
+	distributedWrites         int
+	mdmCommandsReceived       int
+	distributedReads          int
+	configRequests            int
+	configErrors              int
+	resultLogRequests         int
+	orbitErrors               int
+	mdmErrors                 int
+	ddmDeclarationItemsErrors int
```

Comment:
It can be useful during load-testing to have some stats here that sanity-checks that things are working as expected, e.g. like the `mdmSessions` and `mdmCommandsReceived`, stats that count normal behavior and not just errors. Not sure what such a stat would look like for ddm (I think it would still be counted in `mdmSessions` and `mdmCommandsReceived`), but maybe counting configurations, activations and statuses?